### ISSUE TITLE
Jingle mode not working

### ISSUE THREAD

**Issue by themadcap1976** [AzuraCast/AzuraCast#5573]
### Installation Method

One-Click Installation (Docker)

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

v0.17.1 Stable • Docker • PHP 8.1

### What happened?

Playlists set to jingle mode still display artist song name.  This is the same across different stations.

### Relevant log output

```shell
0 [lang:3] API nextsong - Sending POST request to 'http://127.0.0.1:6010/api/internal/2/liquidsoap/nextsong' with body: 
2022/07/05 21:48:30 [lang:3] API nextsong - Response (200): annotate:title="1 you are listening to marina rock radio",duration="6.00",song_id="6a35ae125a378c1b1c23dc73057cdc0b",media_id="1857",liq_amplify="0.00dB",jingle_mode="true":media:Bumpers/1_you_are_listening_to_marina_rock_radio.mp3
[mp3 @ 0x7f383c047180] Estimating duration from bitrate, this may be inaccurate
[mp3 @ 0x7f383c047180] Estimating duration from bitrate, this may be inaccurate
2022/07/05 21:48:30 [cross_0:3] Analysis: -111.785923dB / -34.971274dB (4.96s / 4.96s)
2022/07/05 21:48:31 [lang:3] API feedback - Sending POST request to 'http://127.0.0.1:6010/api/internal/2/liquidsoap/feedback' with body: {
2022/07/05 21:48:31 [lang:3]   "song_id": "37ef3bf65defa5a5084cf572c8c489d5",
2022/07/05 21:48:31 [lang:3]   "is_live": false,
2022/07/05 21:48:31 [lang:3]   "playlist_id": "12",
2022/07/05 21:48:31 [lang:3]   "media_id": "1939"
2022/07/05 21:48:31 [lang:3] }
2022/07/05 21:48:31 [lang:3] API feedback - Response (200): true
[mp3float @ 0x7f38483e7e80] Could not update timestamps for discarded samples.
```

**Comment by SlvrEagle23** [comment]
@themadcap1976 Try updating to 0.17.2 stable, which was just released this week. This was among the many bug fixes included in the latest version.

**Comment by pimseb** [comment]
I've updated to 0.17.2 stable and I confirm that the jingle mode isn't working : artist/title is still being sent 
```
2022/07/07 09:43:59 [lang:3] API nextsong - Response (200): annotate:title="sweeper masculin 1",artist="Jingles",duration="9.00",song_id="48d8663976a88f4ad4f0a6bf5e08469d",media_id="10569",liq_amplify="0.00dB",liq_cross_duration="0.00",liq_fade_in="2.00",liq_fade_out="0.30",liq_cue_out="2.70",jingle_mode="true":media:JINGLES/sweeper_masculin_1.mp3
2022/07/07 09:44:10 [lang:3] API feedback - Sending POST request to 'http://127.0.0.1:6010/api/internal/1/liquidsoap/feedback' with body: { "artist": "", "title": "" }
2022/07/07 09:44:10 [lang:3] API feedback - Response (200): true
2022/07/07 09:44:11 [lang:3] API feedback - Sending POST request to 'http://127.0.0.1:6010/api/internal/1/liquidsoap/feedback' with body: {
2022/07/07 09:44:11 [lang:3]   "song_id": "d9fcdbff24c872f9b5ea0e58e75e69ec",
2022/07/07 09:44:11 [lang:3]   "playlist_id": "13",
2022/07/07 09:44:11 [lang:3]   "media_id": "3918"
2022/07/07 09:44:11 [lang:3] }
2022/07/07 09:44:11 [lang:3] API feedback - Response (200): true
```

I think that this behaviour appeared in the version 0.17.0

**Comment by pimseb** [comment]
Hmm yeah, true. Here is a longer log view

```
2022/07/07 10:28:11 [lang:3] API nextsong - Response (200): annotate:title="Who Do You Love Now? (Stringer) (Radio Version) (2001)",artist="Riva feat. Dannii Minogue",duration="202.00",song_id="22646b918f563e4786cf00b4ce353b66",media_id="1740",liq_amplify="0.00dB",liq_cross_duration="0.00",liq_fade_in="2.00",liq_fade_out="2.00",playlist_id="13":media:200x-TOP/Riva Feat. Dannii Minogue - Who Do You Love Now (Stringer) (Radio Version) (2001).mp3
2022/07/07 10:28:11 [decoder.id3:2] Error while decoding file tags: CamomileLibrary__CharEncoding.Configure(Config).Malformed_code
2022/07/07 10:28:15 [decoder:2] Decoding "/var/azuracast/stations/tdr/media/JINGLES/General/sweeper_fr_3.mp3" ended: Ffmpeg_decoder.End_of_file.
2022/07/07 10:28:16 [next_song:3] Prepared "/var/azuracast/stations/tdr/media/200x-TOP/Riva Feat. Dannii Minogue - Who Do You Love Now (Stringer) (Radio Version) (2001).mp3" (RID 312).
[mp3float @ 0x7f110032b200] Could not update timestamps for skipped samples.
2022/07/07 10:28:16 [cue_next_song:3] Cueing in...
2022/07/07 10:28:16 [lang:3] API nextsong - Sending POST request to 'http://127.0.0.1:6010/api/internal/1/liquidsoap/nextsong' with body: 
2022/07/07 10:28:16 [cross_0:3] Analysis: -14.406729dB / -53.242926dB (0.00s / 0.04s)
2022/07/07 10:28:16 [crossfade_0:3] Simple transition: crossed, fade-in, fade-out.
2022/07/07 10:28:16 [lang:3] API nextsong - Response (200): annotate:title="The Devil's Holding On (2022)",artist="Blasterjaxx feat. Diandra Faye",duration="186.00",song_id="5ea6f086469beb49ef5ab51ecb257b38",media_id="13197",liq_amplify="0.00dB",liq_cross_duration="0.00",liq_fade_in="2.00",liq_fade_out="2.00",playlist_id="6":media:NEWS/Blasterjaxx feat. Diandra Faye - The Devil's Holding On (2022).mp3
2022/07/07 10:28:23 [metadata.map_10:3] Inserting missing metadata.
2022/07/07 10:28:23 [lang:3] API feedback - Sending POST request to 'http://127.0.0.1:6010/api/internal/1/liquidsoap/feedback' with body: { "artist": "", "title": "" }
2022/07/07 10:28:23 [lang:3] API feedback - Response (200): true
2022/07/07 10:28:28 [metadata.map_10:3] Inserting missing metadata.
2022/07/07 10:28:28 [lang:3] API feedback - Sending POST request to 'http://127.0.0.1:6010/api/internal/1/liquidsoap/feedback' with body: {
2022/07/07 10:28:28 [lang:3]   "song_id": "22646b918f563e4786cf00b4ce353b66",
2022/07/07 10:28:28 [lang:3]   "playlist_id": "13",
2022/07/07 10:28:28 [lang:3]   "media_id": "1740"
2022/07/07 10:28:28 [lang:3] }
2022/07/07 10:28:28 [lang:3] API feedback - Response (200): true
```
As you see there is a strange behaviour at 10:28:23 : this  ` { "artist": "", "title": "" }` with empty strings is obviously a 5 seconds jingle, and 5 seconds after at 10:28:28 the artist/title of a music file is sent

**Comment by fidegr** [comment]
> Hello, after updating to version 17.2 we are also having this error. The jingles are not playing.

After seeing that the jingles did not work, I tried to activate the option "hide metadata from listeners" and the jingle playlists have started to work. It seems that the failure occurs when this option is not checked.

**Comment by pimseb** [comment]
Concerning my problem, I had the crossfade setting set to "disabled" in the station settings page, but a custom value in the manual liquidsoap configuration to enhance the crossfade for my taste :
`radio = crossfade(smart=false, duration=2.,fade_out=2.7,fade_in=0.2,radio)`

I've set the crossfade setting to "Normal Mode" in the station settings page and now the jingle mode is working at it should (no metadata is sent anymore). I don't understand  how this crossfade settings interfere with the jingle mode ?!
But there is now another problem : the crossfade between songs is blocked on 0.00 (no crossfade) even if I try to change to smart mode, or change the crossfade duration. So I had to reenter manually my previous line in the liquidsoap conf file.
